components/benefit_sponsors/app/models/effective/datatables/benefit_sponsors_employer_datatable.rb

```diff
@@ -282,6 +282,11 @@ def nested_filter_definition
         end
         filters
       end
+
+      def authorized?(current_user, _controller, _action, _resource)
+        return true if current_user&.has_hbx_staff_role?
```

Comment:
```suggestion
        current_user&.has_hbx_staff_role?
```

Reply:
current_user&.has_hbx_staff_role? will return true/false/nil. I made that more specific to true/false .